conf.d/cassandra.yaml.example

```diff
@@ -21,54 +21,81 @@ init_config:
   # Read http://docs.datadoghq.com/integrations/java/ to learn how to customize it
   conf:
     - include:
-        domain: org.apache.cassandra.db
+        domain: org.apache.cassandra.metrics
+        type: ClientRequest
+        scope:
+          - Read
+          - Write
+        name:
+          - Latency
+          - Timeouts
+          - Unavailables
         attribute:
+          - Count
+          - OneMinuteRate
+    - include:
+        domain: org.apache.cassandra.metrics
+        type: ClientRequest
+        scope:
+          - Read
+          - Write
+        name:
+          - TotalLatency
+    - include:
+        domain: org.apache.cassandra.metrics
+        type: Storage
+        name:
+          - Load
+          - Exceptions
+    - include:
+        domain: org.apache.cassandra.metrics
+        type: ColumnFamily
+        name:
+          - TotalDiskSpaceUsed
           - BloomFilterDiskSpaceUsed
           - BloomFilterFalsePositives
           - BloomFilterFalseRatio
-          - Capacity
-          - CompletedTasks
           - CompressionRatio
-          - ExceptionCount
-          - Hits
-          - KeyCacheRecentHitRate
           - LiveDiskSpaceUsed
           - LiveSSTableCount
-          - Load
           - MaxRowSize
           - MeanRowSize
           - MemtableColumnsCount
-          - MemtableDataSize
+          - MemtableLiveDataSize
           - MemtableSwitchCount
           - MinRowSize
-          - PendingTasks
-          - RangeOperations
-          - ReadCount
-          - ReadOperations
-          - RecentHitRate
-          - RecentRangeLatencyMicros
-          - RecentReadLatencyMicros
-          - RecentWriteLatencyMicros
-          - Requests
-          - RowCacheRecentHitRate
-          - Size
-          - TotalDiskSpaceUsed
-          - TotalRangeLatencyMicros
-          - TotalReadLatencyMicros
-          - TotalWriteLatencyMicros
-          - UpdateInterval
-          - WriteCount
-          - WriteOperations
       exclude:
-        keyspace: system
+        keyspace:
+          - system
+          - system_auth
+          - system_distributed
+          - system_traces
```

Comment:
Thanks for your feedback on it @sethrosenblum. I am adding it to the list of exclusion.